frontend/src/components/register/RegisterInfo.js

```diff
@@ -8,12 +8,34 @@ import { ReactComponent as Rising } from "../../assets/growth-curve-cuate.svg"
 
 const useStyles = makeStyles(theme => ({
 	svg: {
-		position: 'absolute',
-		bottom: 0
+		bottom: 0,
+		marginBottom: "2rem",
+		left: 0,
+		marginLeft: "6rem",
```

Comment:
The image should have no margin as that is the intended effect.

Reply:
Only on the Register Page svg?

Reply:
If so changed

Reply:
```
`	svg: {
		position:"absolute",
		bottom: 0,
		left: 0,
		[theme.breakpoints.down(820)]:
		{
			height: 500,
			marginBottom: "-4rem",
			marginLeft: "-2rem",
			position:"static",
		},`
```